src/composables/useRenderer/index.ts

```diff
@@ -274,13 +250,33 @@ export function useRenderer(
     }
   })
 
+  // Register loop
```

Comment:
For separation of concerns, I'd move this somewhere else. 

Almost all core code has access to the renderer, so this could live just about anywhere.

On the other hand, `useRenderer` has to be passed `render`, `loop`, `invalidate`, `advance`, in order to do `loop.register`.